front/lib/resources/conversation_resource.ts

```diff
@@ -188,16 +188,22 @@ export class ConversationResource extends BaseResource<ConversationModel> {
     return mentions;
   }
 
-  static async listAllBeforeDate(date: Date): Promise<ConversationResource[]> {
-    const conversations = await this.model.findAll({
-      where: {
-        updatedAt: {
-          [Op.lt]: date,
+  static async listAllBeforeDate(
+    auth: Authenticator,
+    date: Date
+  ): Promise<ConversationResource[]> {
+    const conversations = await this.baseFetch(
+      auth,
+      { includeDeleted: true, includeTest: true },
```

Comment:
just to note: in that specific case the `includeTest` is not needed, but I like it here for futur baseFetch refactor.